hue/verify_hue_running.py

```diff
@@ -0,0 +1,88 @@
+"""verify_hue_running.py: Script for Hue initialization action test.
+"""
+
+import requests
+import socket
+import subprocess
+
+BASE = 'localhost'
+PORT = 8888
+
+
+class HueApi(object):
```

Comment:
Similar feedback on the design of this class as in GangliaApi -- getter methods should not mutate the state of the object. Consider making these *Api classes stateless, and make assertions in other functions